From 9b31f5cbc3479abb82b129d2caeba6a3a85f8617 Mon Sep 17 00:00:00 2001
From: Ben <Ben.Cownley@arm.com>
Date: Thu, 1 Aug 2024 17:25:37 +0100
Subject: [PATCH] arm-systemready/documentation: Enable SIE extension of ACS

Update the user document for enabling the SIE testing of SystemReady IR
ACS.

Upstream-Status: Pending

Signed-off-by: Ben Cownley <ben.cownley@arm.com>
Signed-off-by: Michael Zhao <michael.zhao2@arm.com>
---
 meta-arm-systemready/README.md | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/meta-arm-systemready/README.md b/meta-arm-systemready/README.md
index 49cfed6e..03602456 100644
--- a/meta-arm-systemready/README.md
+++ b/meta-arm-systemready/README.md
@@ -117,6 +117,9 @@ To reuse the supported Arm SystemReady IR on other machines, you will need to:
 2. Set the `ARM_SYSTEMREADY_ACS_CONSOLE` variable at the MACHINE configuration
    level for running the ACS tests in the bitbake `testimage` task. Also refer
    to the configuration file of the [`fvp-base`][] machine.
+3. SIE testing is an optional extension of the ACS testing.
+   Should the platform require it, add the `acs-sie` flag to the
+   `MACHINE_FEATURES` variable at the machine configuration level.
 
 ### Extend
 
-- 
2.34.1

